pyproject.toml

```diff
@@ -1,10 +1,35 @@
 [build-system]
+requires = [
+         "setuptools",
+         "setuptools_scm",
+         "wheel",
+         "oldest-supported-numpy",
```

Comment:
Do we actually have a build time requirement on numpy?

Reply:
No.

Reply:
delete it!!

Reply:
Done.